jetpack.php

```diff
@@ -154,11 +167,14 @@ function jetpack_admin_missing_autoloader() { ?>
 	return;
 }
 
-// Load all the packages.
+/**
+ * Load all the packages.
+ *
+ * We want to fail gracefully if `composer install` has not been executed yet, so we are including the autoloader.
```

Comment:
"so we are including the autoloader." is this correct?

Reply:
In the sense of we are including, not requiring, the autoloader in order to allow us to do magic instead of having the php fatal occur.

Open to better wording. Since this is me being a bit creative, I'd rather err on the side of over-explanation inline.